docs/source/40_deploying_ADAM.md

```diff
@@ -86,46 +82,52 @@ The typical flow of data to and from your ADAM application on EC2 will be:
 - Compute with ADAM, write output to HDFS
 - Copy data you wish to persist for later use to S3
 
-For small test files you may wish to skip S3 by  uploading directly to 
-spark-master using `scp` and then copy to HDFS using 
-`hadoop fs -put sample1.bam /datadir/` 
+For small test files you may wish to skip S3 by uploading directly to 
+spark-master using `scp` and then copying to HDFS using:
+```
+hadoop fs -put sample1.bam /datadir/
+```
 
-From ADAM shell, or as parameter to ADAM submit, you would refer HDFS URLs
-such as:
+From the ADAM shell, or as a parameter to ADAM submit, you would refer to HDFS URLs like this:
 ```
 adam-submit transformAlignments hdfs://spark-master/work_dir/sample1.bam \
                       hdfs://spark-master/work_dir/sample1.adam
 ```
 
 #### Bulk Transfer between HDFS and S3
+
 To transfer large amounts of data back and forth from S3 to HDFS, we suggest using 
 [Conductor](https://github.com/BD2KGenomics/conductor).
 It is also possible to directly use AWS S3 as a distributed file system, 
 but with some loss of performance.
 
 #### Terminate Cluster
+
 Shutdown the cluster using:
 ```
 cgcloud terminate-cluster -c cluster1 spark
 ```
 
 #### CGCoud options and Spot Instances
-View help docs for all options of the the `cgcloud create-cluster` command:
+
+View help docs for all options of the `cgcloud create-cluster` command:
 ```
 cgcloud create-cluster -h
```

Comment:
For me, if the bash code block shows one or commands that the user would be likely to copy and paste into a bash shell, then I typically don't include the $.
```
mvn install
./bin/adam-submit transformAlignments foo.bam bar.adam
spark-submit conductor-0.5-SNAPSHOT-distribution.jar hdfs://bar.adam s3://bar.adam
```

If the code block is showing the output of a command, then I would include the $.
```
$ adam-submit --help
Using ADAM_MAIN=org.bdgenomics.adam.cli.ADAMMain
Using SPARK_SUBMIT=/usr/local/bin/spark-submit

       e         888~-_          e             e    e
      d8b        888   \        d8b           d8b  d8b
     /Y88b       888    |      /Y88b         d888bdY88b
    /  Y88b      888    |     /  Y88b       / Y88Y Y888b
   /____Y88b     888   /     /____Y88b     /   YY   Y888b
  /      Y88b    888_-~     /      Y88b   /          Y888b

Usage: adam-submit [<spark-args> --] <adam-args>
...
```